tracer/src/Datadog.Tracer.Native/calltarget_tokens.h

```diff
@@ -78,6 +80,7 @@ class CallTargetTokens
 public:
     mdTypeRef GetObjectTypeRef();
     mdTypeRef GetExceptionTypeRef();
+    mdTypeRef GetBubbleUpExceptionTypeRef();
```

Comment:
This is also something that should be placed into `TracerTokens` (`tracer_tokens.cpp`) because it's specific to tracer instrumentation, rather than all calltarget instrumentation(s).

Reply:
fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba